module/batch-action/migrations/Version20201102124000.php

```diff
@@ -30,12 +30,12 @@ public function up(Schema $schema): void
 
         $this->addSql(
             'CREATE TABLE batch_action_entry (
-                      id UUID NOT NULL, 
                       batch_action_id UUID NOT NULL,
+                      entry_id UUID NOT NULL, 
```

Comment:
in `batch_action` we have resource type. Maybe it should be resource_id - it identifies the resource. `batch_action_entry` is the name because it's an entry of entire batch set